tests/helpers.ts

```diff
@@ -78,3 +79,15 @@ export const balenaAPIMock = () => {
 			configVarSchema: [],
 		});
 };
+
+export const cleanOutput = (output: string[] | string) => {
```

Comment:
@pdcastro BTW, what's your preferred style for function declaration when it's directly exported, such as here?

Reply:
I would use the `function` keyword here as well:
```
export function cleanOutput(output: string[] | string) {
```